candi/bashible/bashbooster/60_deckhouse.sh

```diff
@@ -72,3 +72,12 @@ bb-deckhouse-get-disruptive-update-approval() {
     bb-log-info "Disruption approved!"
     bb-flag-set disruption
 }
+
+bb-is-ubuntu-version?() {
```

Comment:
I didn't get the idea of moving bb-is-ubuntu-version here. It is more fittable to leave it in 55_ubuntu_version or just 55_ubuntu.

Reply:
lets move.